### PR TITLE
Feature/upload files

### PR DESCRIPTION
**Pull request by GabrielMoreschi**
Foi criado o componente visual de upload de arquivos do módulo documental.

### CODE REVIEW

File: components/document-dialog/DocumentUploadDialog.vue [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                    </v-btn>
+                </v-card-actions>
+            </v-card>
+        </v-dialog>
+    </div>
+</template>
+<i18n>
+    {
+        "en": {
+            "date-label": "Date of Document",
```
Comment (guigawiller) [inline components/document-dialog/DocumentUploadDialog.vue:74]: Check if this is properly written.

File: package.json [hunk 2]
```diff
@@ -41,6 +41,7 @@
         "shpjs": "^4.0.2",
         "v-mask": "^2.2.3",
         "vue-chartjs": "^3.5.1",
+        "vue-resource": "^1.5.3",
```
Comment (guigawiller) [inline package.json:44]: Is this really necessary?

File: store/document.js [hunk 3]
```diff
@@ -189,6 +165,7 @@ export const actions = {
         commit('setFilterOptions', data)
     },
 
+
```
Comment (guigawiller) [inline store/document.js:168]: Remove blank line

**Review by guigawiller — CHANGES_REQUESTED** [review]

File: components/document-dialog/DocumentUploadDialog.vue (hunk 1 above)
Comment (GabrielMoreschi) [reply components/document-dialog/DocumentUploadDialog.vue:74]: Resolved

File: store/document.js (hunk 3 above)
Comment (GabrielMoreschi) [reply store/document.js]: Resolved

File: package.json (hunk 2 above)
Comment (GabrielMoreschi) [reply package.json]: No.
Resolved.

File: components/document-dialog/DocumentUploadDialog.vue [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+                            <input type="file" ref="fileInput" />
+                            <v-icon> mdi-paperclip </v-icon>
+                        </v-btn>
+                    </v-col>
+                </v-card-actions>
+                <v-divider></v-divider>
+                <v-card-actions>
+                    <v-spacer></v-spacer>
+                    <v-btn color="secondary" text @click="dialog = false">
+                        Cancelar
```
Comment (guigawiller) [inline components/document-dialog/DocumentUploadDialog.vue:61]: Try putting this in the i18n pattern.

File: components/document-dialog/DocumentUploadDialog.vue [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+                    </v-btn>
+                    <v-btn color="secondary" text @click="save()">
+                        Salvar
```
Comment (guigawiller) [inline components/document-dialog/DocumentUploadDialog.vue:64]: Try putting this in the i18n pattern.

**Review by guigawiller — CHANGES_REQUESTED** [review]

File: components/document-dialog/DocumentUploadDialog.vue (hunk 5 above)
Comment (GabrielMoreschi) [reply components/document-dialog/DocumentUploadDialog.vue]: Resolved

**Review by guigawiller — APPROVED** [review]

**Pull request merged**